packages/fxa-settings/src/pages/Legal/index.tsx

```diff
@@ -20,7 +20,10 @@ const Legal = (_: RouteComponentProps) => {
       <CardHeader headingTextFtlId="legal-header" headingText="Legal" />
 
       <div className="flex justify-between text-sm">
-        <Link className="link-blue" to="/legal/terms">
+        <Link
```

Comment:
This is happening because with such little content on the page, the `card` class on `AppLayout` doesn't reach the max-width size. If we had a paragraph or a longer heading on this page this wouldn't be a problem.

We could've also vertically stacked these links instead but I matched the current design.